### DISCUSSION TITLE
How to make tasmota as mobus TCP  for read data from mobus TCP  client?

### DISCUSSION THREAD

**Discussion by Rojer7084** [arendst/Tasmota#12963] [Q&A] [answered]
Hi friend 
    Can anyone  use  tasmota  as mobus TCP  for read data from mobus TCP  client. ?  if   you use to do it , please recommence 
many thanks

**Comment by barbudor** [comment, marked as answer]
Tasmota can connect to a serial RS485 MODBUS device to parse messages such as energy meter.
It is great to interface non IP devices with the IP world in general and MQTT especially

However, connecting to MODBUS-over-IP is not supported.

Frankly, I don't see much the point of using tasmota to connect to an IP device and bridging MODBUS/IP to MQTT for example.
This can most efficiently be supported directly in the home automation system.

**Reply by sfromis** [reply]
I'd probably look to use Node-Red for such, maybe via a package like [node-red-contrib-modbus](https://flows.nodered.org/node/node-red-contrib-modbus). Not had the need.

**Reply by Rojer7084** [reply]
so many thanks

**Comment by Rojer7084** [comment]
so many thanks for your reply

ในวันที่ พ. 25 ส.ค. 2021 เวลา 00:03 sfromis ***@***.***>
เขียนว่า:
